### PR TITLE
Added History to Readme

### PR DESCRIPTION
**Pull request by sampoearas**
Added History and Supported Versions to file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,20 @@
...
+--------|--------
+[v1.0.1](https://github.com/ArasLabs/aras-homepage/releases/tag/v1.0.1) | Tested on IE, Edge, Firefox 60 ESR, Chrome.
+[v1.0.0](https://github.com/ArasLabs/aras-homepage/releases/tag/v1.0.0) | First release. Tested on Internet Explorer 11, Firefox 38 ESR, Chrome.
+
+#### Supported Aras Versions
+
+Project | Aras
+--------|------
+[v1.0.1](https://github.com/ArasLabs/aras-homepage/releases/tag/v1.0.1) | All Aras Versions
+[v1.0.0](https://github.com/ArasLabs/aras-homepage/releases/tag/v1.0.0) | All Aras Versions; Old Community Board Migration
```
Comment (EliJDonahue) [inline README.md]: This project wasn't a migration from the old community site.
```suggestion
[v1.0.0](https://github.com/ArasLabs/aras-homepage/releases/tag/v1.0.0) | All Aras Versions
```

File: README.md [hunk 2]
```diff
@@ -2,7 +2,20 @@
...
+Release | Notes
+--------|--------
+[v1.0.1](https://github.com/ArasLabs/aras-homepage/releases/tag/v1.0.1) | Tested on IE, Edge, Firefox 60 ESR, Chrome.
+[v1.0.0](https://github.com/ArasLabs/aras-homepage/releases/tag/v1.0.0) | First release. Tested on Internet Explorer 11, Firefox 38 ESR, Chrome.
+
+#### Supported Aras Versions
+
+Project | Aras
+--------|------
+[v1.0.1](https://github.com/ArasLabs/aras-homepage/releases/tag/v1.0.1) | All Aras Versions
```
Comment (EliJDonahue) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -2,7 +2,20 @@
 
 This project sets up a "homepage" that lists the Aras Innovator instances installed on a server. The list contains links to each instance's login page and Nash page. The homepage acts as a directory for easy access to your Aras Innovator installations.
 
-Works for all versions of Aras Innovator.
+## History
+
+Release | Notes
+--------|--------
+[v1.0.1](https://github.com/ArasLabs/aras-homepage/releases/tag/v1.0.1) | Tested on IE, Edge, Firefox 60 ESR, Chrome.
```
Comment (EliJDonahue) [inline README.md]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -2,7 +2,20 @@
...
+[v1.0.0](https://github.com/ArasLabs/aras-homepage/releases/tag/v1.0.0) | First release. Tested on Internet Explorer 11, Firefox 38 ESR, Chrome.
```
Comment (EliJDonahue) [inline README.md]:
```suggestion
[v1.0.0](https://github.com/ArasLabs/aras-homepage/releases/tag/v1.0.0) | First release. Tested on Internet Explorer 11, Firefox, Chrome.
```

**Review by EliJDonahue — CHANGES_REQUESTED** [review]
We don't need a v1.0.1 for this project. Just the base release is fine until we add new functionality.

File: README.md [hunk 5]
```diff
@@ -2,7 +2,18 @@
 
 This project sets up a "homepage" that lists the Aras Innovator instances installed on a server. The list contains links to each instance's login page and Nash page. The homepage acts as a directory for easy access to your Aras Innovator installations.
 
-Works for all versions of Aras Innovator.
+## History
+
+Release | Notes
+--------|--------
+[v1.0.0](https://github.com/ArasLabs/aras-homepage/releases/tag/v1.0.0) | First release. Tested on Internet Explorer 11, Firefox, Chrome.
```
Comment (sampoearas) [inline README.md]:
```suggestion
[v1.0.0](https://github.com/ArasLabs/aras-homepage/releases/tag/v1.0.0) | First release. Tested on Internet Explorer, Edge, Firefox, Chrome.
```

**Review by sampoearas — COMMENTED** [review]
Additional Suggestion

File: README.md (hunk 5 above)
Comment (EliJDonahue) [reply README.md]: I'd say you can leave out the Firefox version. This project is pretty generic, so the behavior from one release of a browser to the next will be pretty stable.

Comment (sampoearas) [reply README.md]: Updated to addition of Edge Only

**Pull request merged**